src/controller/AdminController.php

```diff
@@ -102,7 +103,7 @@ public function displayPendingUsers(UserModel $userModel, PostModel $postModel,
 		$pendingUsers = $userModel->loadPendingUsers();
 
 		// display 
-		$this->render('admin_pending_users.twig', $_SESSION, $pendingUsers);
+		$this->render('admin_pending_users.twig', [], $pendingUsers, [], [], [], $_SESSION);
```

Comment:
```suggestion
		$this->render('admin_pending_users.twig', $_SESSION, $pendingUsers);
```

Reply:
Modif Ok : 
$this->render('admin_pending_users.twig', $_SESSION, [], $pendingUsers, [], [], []);